### PR TITLE
LVRUBY-109: Added images to news

### PR DESCRIPTION
**Pull request by kmandziy**
## JIRA ticket

_Please provide the URL for the JIRA ticket._

## Code reviewers

- [ ] @omaik 
- [x] @PidtserkovnyiVitalii 

## Summary of issue

Added ability to upload images to news

## Summary of change

Added filed image to News
Added image uploader to news.rb
Displaying News images
Screenshot from 2019-05-16 16-19-23
Screenshot from 2019-05-16 16-40-26

## Review checklist

- [ ] Rubocop doesn't raise any warnings and offenses
- [ ] New features are covered with new specs
- [ ] All specs pass

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

File: app/assets/stylesheets/pages/news.scss [hunk 1]
```diff
@@ -24,6 +24,31 @@
...
+  float: left;
+  border-radius: 50px;
+  margin-right: 5px;
+  width: 50px;
+  height: 50px;
+  overflow:hidden;
+
+  img {
+    display:block;
+    width:100%;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

File: app/assets/stylesheets/pages/news.scss [hunk 2]
```diff
@@ -24,6 +24,31 @@
...
+  }
+}
+
+.thumbnail-social-icon {
+  float: left;
+  border-radius: 50px;
+  margin-right: 5px;
+  width: 50px;
+  height: 50px;
+  overflow:hidden;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

File: app/assets/stylesheets/pages/news.scss [hunk 3]
```diff
@@ -24,6 +24,31 @@
...
+
+  img {
+    display:block;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

File: app/assets/stylesheets/pages/news.scss [hunk 4]
```diff
@@ -24,6 +24,31 @@
...
+    width:100%;
+  }
+}
+
+.thumbnail-social-icon {
+  float: left;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

File: app/assets/stylesheets/pages/news.scss [hunk 5]
```diff
@@ -24,6 +24,31 @@
...
 }
 
+.thumbnail-news {
+  height: 300px;
+  margin: 0;
+  overflow:hidden;
+
+  img {
+    display:block;
+    width:100%;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

File: app/assets/stylesheets/pages/news.scss [hunk 6]
```diff
@@ -24,6 +24,31 @@
   }
 }
 
+.thumbnail-news {
+  height: 300px;
+  margin: 0;
+  overflow:hidden;
+
+  img {
+    display:block;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

File: app/assets/stylesheets/pages/news.scss [hunk 7]
```diff
@@ -24,6 +24,31 @@
   }
 }
 
+.thumbnail-news {
+  height: 300px;
+  margin: 0;
+  overflow:hidden;
```
[the remark this answers is not in the document]
Comment (kmandziy) [reply app/assets/stylesheets/pages/news.scss]: Added space

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

File: app/uploaders/image_uploader.rb [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```
Comment (omaik) [inline app/uploaders/image_uploader.rb]: Seems that you forget to clean - https://github.com/dark-side/ai-collaboration-platform/blob/LVRUBY-109/app/uploaders/avatar_uploader.rb

File: app/controllers/news_controller.rb [hunk 9]
```diff
@@ -50,7 +50,7 @@ def destroy
   private
 
   def news_params
-    params.require(:news).permit(:title, :short_information, :body)
+    params.require(:news).permit(:title, :short_information, :body, :image)
```
Comment (omaik) [inline app/controllers/news_controller.rb]: Above in controller - `@news.update_column(:views, @news.views + 1)` Please update using `increment` - https://apidock.com/rails/ActiveRecord/Persistence/increment

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

File: app/controllers/news_controller.rb (hunk 9 above)
Comment (kmandziy) [reply app/controllers/news_controller.rb]: Already changed

File: app/uploaders/image_uploader.rb (hunk 8 above)
Comment (kmandziy) [reply app/uploaders/image_uploader.rb]: Removed

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.38%** covered at **4.51** hits/line

**Pull request merged**